packages/manager/src/dev-tools/FeatureFlagTool.tsx

```diff
@@ -34,6 +34,7 @@ const options: { flag: keyof Flags; label: string }[] = [
   { flag: 'linodeInterfaces', label: 'Linode Interfaces' },
   { flag: 'lkeEnterprise', label: 'LKE-Enterprise' },
   { flag: 'mtc2025', label: 'MTC 2025' },
+  { flag: 'nodebalancerIpv6', label: 'NodeBalancer Dual Stack Support' },
```

Comment:
```suggestion
  { flag: 'nodebalancerIpv6', label: 'NodeBalancer Dual Stack (IPv6)' },
```
I think it would be beneficial to try and keep the name in the feature flag list and the flag itself similar, if not the same. This helps with consistency for reviewers, anyone needing to toggle the feature flag, etc.